src/routes/birthday/retrieve/byDateAndTimezone.ts

```diff
@@ -61,11 +51,9 @@ export class UserRoute extends Route {
         );
 
         if (results.length === 0) {
-            response.statusMessage = 'No Birthdays found on that Date and Timezone';
-            return response.status(200).json({ amount: 0, birthdays: [], error: 'No Birthdays found on that Date and Timezone' });
+            return response.badRequest({ amount: 0, birthdays: [], error: 'No Birthdays found on that Date and Timezone' });
```

Comment:
Same question as in https://github.com/BirthdayyBot/BirthdayyBotSapphire/pull/97/files#r1137277354